frontend/containers/HomePage/HomePageNew.tsx

```diff
@@ -0,0 +1,32 @@
+import CardBlock from "@/components/Blocks/CardsBlock/CardBlock";
+import HeroBlock from "@/components/Blocks/HeroBlock/HeroBlock";
+import TitleBlock from "@/components/Blocks/TitleBlock/TitleBlock";
+import TextAndMedia from "@/components/TextAndMedia/TextAndMedia";
+
+import styles from "./HomePage.module.css";
+
+const HomePageNew = ({ homepage }: { homepage: HomePage[] }) => {
+  return (
+    <div className={styles[""]}>
+      {HomePage?.map((content, _index) => {
```

Comment:
```suggestion
      {homepage?.map((content, _index) => {
```

Reply:
Thanks. This was a first set-up for a new homepage (as the current one is still hardcoded). So this isn't actually used yet. But I've altered the code.